system-variables.md

```diff
@@ -88,11 +132,17 @@ SET  GLOBAL tidb_distsql_scan_concurrency = 10;
 >
 > Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.
 
-### interactive_timeout
+### port
 
-- Scope: SESSION | GLOBAL
-- Default value: 28800
-- This variable represents the idle timeout of the interactive user session, which is measured in seconds. Interactive user session refers to the session established by calling [`mysql_real_connect()`](https://dev.mysql.com/doc/c-api/5.7/en/mysql-real-connect.html) API using the `CLIENT_INTERACTIVE` option (for example, MySQL shell client). This variable is fully compatible with MySQL.
+- Scope: NONE
+- Default value: 4000
+- The port that the `tidb-server` is listening on when speaking the MySQL protocol.
```

Comment:
What does "speaking" mean?

Reply:
TiDB has a HTTP (10080) and a MySQL port (4000). Clients are said to "talk to the server", the network protocol the server speaks is "MySQL". This is different from SQL, which happens at a higher level. Talking/speaking is the basic protocol negotiation.